features/vendor_views_closed_auction.feature

```diff
@@ -13,6 +13,8 @@ Feature: Vendor views closed auctions
     And I should see when the auction started
     And I should see when the auction ended
     And there should be meta tags for the closed auction
+    And an email notification is sent to the winning bidder
+    And I should receive an email notifying me that I won
```

Comment:
I don't love that we need to use `And an email notification is sent to the winning bidder`. It's sort of like saying "And the sun rises the next day / And I should see a thing on my screen" ... the feature specs, I _think_ should assume the standard passage of time.

So maybe Timecop helps us avoid this?